test/unit/test_indirect_loop.py

```diff
@@ -124,6 +124,7 @@ def test_mismatching_indset(self, iterset, x):
             op2.par_loop(op2.Kernel("", "dummy"), iterset,
                          x(op2.WRITE, op2.Map(iterset, op2.Set(nelems), 1)))
 
+    @pytest.mark.skip("allow different block size")
```

Comment:
Just remove this test, it does not apply anymore.

Reply:
Cool.